salt/metalk8s/repo/online.sls

```diff
@@ -1,6 +1,6 @@
 {%- from "metalk8s/map.jinja" import repo with context %}
 
-Configure EPEL repository:
+Configure epel repository:
   pkg.installed:
```

Comment:
this one too

Reply:
Not sure this is relevant to use it here since this macro is to hide the magic about offline/online differences.